index.html

```diff
@@ -10,228 +10,302 @@
             </ul>
-        </nav>
-    </header>
-    <section  class="container">
-        <img src="./images/header-shapes mobile.png" alt="" class="containerimg">
-        <h1 class="title"> I'm Oumaima <br />Glad to see you!</h1>
-        <p class="about-me">
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hesitate to
-            contact me.
-        </p>
-        <h4 class="social-title">LET'S CONNECT</h4>
-        <ul class="social-icons">
-            <li>
-                <a href="#"><img src="./images/social/Twitter.svg" alt="Twitter" /></a>
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Linkedin.svg" alt="Linkedln" /></a>
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Vector-M.svg" alt="Gmail" /></a>
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Github.svg" alt="Github" /></a>
-            </li>
-            <li>
-                <a href="#"> <img src="./images/social/Hand2.svg" alt="Social media"/></a>
-            </li>
-        </ul>    
-    </section>
-                                        <!-- page 2 -->
-                                    <!-- Section 2 : Works -->
-    <section class="work">
-        <ul>
-        <!-- First Card -->
-            <li class="card-container1">
-                <img src="./images/Snapshoot1.png" class="card-container-img1" alt="Snapshoot1">
+            
+            <nav class="navigation">
+                <a class="links" href="#portfolio">Portfolio</a>
+                <a class="links" href="#about">About</a>
+                <a class="links" href="#contact">Contact</a>
+            </nav>
+        </header>
 
-                <h2 class="title-project1">Tonic</h2>  
+        <section class="intro">
 
-                <div class="card-content1">
-                    <ul>
-                        <li class="type1">
-                            <p class="t1">CANOPY</p>
-                            <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
-                            <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
-                        </li>  
-                    </ul>                    
-                </div> 
-                    
-                    <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    
-                    <ul class="tech1">
-                        <li class="language1">html</li>
-                        <li class="language2"> css</li>
-                        <li class="language3">javascript</li>
-                    </ul>
-
-                    <button type="button" class="see-proj1">See Project</button> 
+            <div class="container-global">
```

Comment:
- [ ] While you did a wonderful job implementing the requests from the last reviewer :+1:, as mentioned before you shouldn't have a horizontal scroll in your portfolio, everything should fit the page without scrolling horizontally.
- Below you will find a video in which I show you the differences between the Figma and your design


https://user-images.githubusercontent.com/105117832/221359642-d3a08615-b8c6-43f9-9725-a8f262534862.mp4